app/pages/Onboarding/ImportSeedFlow/index.js

```diff
@@ -23,6 +24,7 @@ const CREATE_PASSWORD = 'CREATE_PASSWORD';
 const ENTRY_STEP = 'ENTRY';
 const OPT_IN_ANALYTICS = 'ANALYTICS';
 const SET_NAME = 'SET_NAME';
+const UPDATE_ACCOUNT_DEPTH = 'UPDATE_ACCOUNT_DEPTH';
```

Comment:
yeah we wouldn't say "update_account_depths" here, even though there are two depth values, it makes more sense to me